projects/swimlane/ngx-ui/src/lib/components/hotkeys/hotkeys.service.ts

```diff
@@ -148,6 +148,7 @@ export function _deregister(comp: any) {
   for (const comb in hotkeys) {
     const hotkeyList = hotkeys[comb];
 
+    // eslint-disable-next-line @typescript-eslint/prefer-for-of
```

Comment:
thoughts: if we do not plan to replace for...loop with for...of then I think we should disable this rule.

Reply:
Disabled in our config... the angualer config adds it back :(